halo/app/ready.go

```diff
@@ -13,39 +13,40 @@ type readinessStatus struct {
 	mu                 sync.RWMutex
 	ConsensusSynced    bool    `json:"consensus_synced"`              // Unhealthy if false
 	ConsensusP2PPeers  int     `json:"consensus_p2p_peers"`           // Unhealthy if 0
+	ConsensusHeight    int64   `json:"consensus_height"`              // Unhealthy if 0
+	ConsensusRunning   bool    `json:"consensus_running"`             // Unhealthy if false
```

Comment:
aded `consensus_running` so the app becomes unhealthy if consensus stalls like on staging. The nodes were healthy even though they stalled